apps/meteor/app/cloud/server/functions/unregisterWorkspace.ts

```diff
@@ -0,0 +1,22 @@
+import { Settings } from '@rocket.chat/models';
+
+import { retrieveRegistrationStatus } from './retrieveRegistrationStatus';
+
+export async function unregisterWorkspace() {
+	const { workspaceRegistered } = retrieveRegistrationStatus();
+	if (!workspaceRegistered) {
+		return true;
+	}
+
+	await Promise.all([
+		Settings.updateValueById('Cloud_Workspace_Id', undefined),
+		Settings.updateValueById('Cloud_Workspace_Name', undefined),
+		Settings.updateValueById('Cloud_Workspace_Client_Id', undefined),
+		Settings.updateValueById('Cloud_Workspace_Client_Secret', undefined),
+		Settings.updateValueById('Cloud_Workspace_Client_Secret_Expires_At', undefined),
+		Settings.updateValueById('Cloud_Workspace_PublicKey', undefined),
+		Settings.updateValueById('Cloud_Workspace_Registration_Client_Uri', undefined),
```

Comment:
```suggestion
		Settings.updateValueById('Cloud_Workspace_Id', null),
		Settings.updateValueById('Cloud_Workspace_Name', null),
		Settings.updateValueById('Cloud_Workspace_Client_Id', null),
		Settings.updateValueById('Cloud_Workspace_Client_Secret', null),
		Settings.updateValueById('Cloud_Workspace_Client_Secret_Expires_At', null),
		Settings.updateValueById('Cloud_Workspace_PublicKey', null),
		Settings.updateValueById('Cloud_Workspace_Registration_Client_Uri', null),
```

Reply:
this is the ONLY PLACE where we use null instead of undefined, type definitions predict undefined, what is the gain/benefit of using null somewhere?

Reply:
I actually found just a SINGLE PLACE using `undefined` 🤔 the is in TS, so it can't anything else.. but clearly there are more usage of `null`..

I don't understand why you trust more on the type definition than on this function.. the point is, `undefined` is ignored and the record is not updated.. so the correct is `null`, the type definition must be changed to reflect this.